build/psgallery-release.yml

```diff
@@ -16,6 +16,7 @@ resources:
 
 variables:
   - group: 'Build Configuration'
+  - group: 'PowerShell Gallery'
```

Comment:
Do we still need this since we use a Service Connection? If not, let's remove the variable group as well